backend/codesirius/problems/serializers/problem.py

```
@@ -1,113 +1,112 @@
+            "executionConstraints",  # READ
+            "sampleTests",  # READ
+            "createdBy",  # READ
+            "status",  # READ
+        ]
+        extra_kwargs = {
+            "status": {"default": Problem.Status.DRAFT, "read_only": True},
+            "id": {"read_only": True},
+        }
+
+    @staticmethod
+    def _extract_related_data(validated_data):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `_extract_related_data`

(ANN205)